### DISCUSSION TITLE
comment changer l'espace alloué à un cours (Chamilo 1.11.26)

### DISCUSSION THREAD

**Discussion by Moigaston** [chamilo/chamilo-lms#5822] [General]
Dans Administration > System Status > Course space, je peux changer l'espace alloué à un cours. Mais ce changement affecte-t'il le cours ou bien seulement le module "documents" - dans lequel le changement de la valeur (en MB) et affiché? Il semble que l'un de mes cours ait atteint sa capacité maximum puisque mes élèves ne peuvent plus envoyer d'exercice ni dans le module "exercices" ni dans le module "travaux" - dans les deux cas il s'agit d'exercices audios (enregistrements à m'envoyer).

**Comment by chatonito** [comment]
- Je ne parle pas français, j'utilise le traducteur Google.
- l'espace attribué au cours sur le serveur depuis l'administration n'affecte pas les fichiers contenus, si votre cours a atteint la limite, augmentez la taille du cours et les fichiers nécessaires peuvent être envoyés, nous avons un cours avec 20G d'espace (vidéos) et si nécessaire il est augmenté.
- si les étudiants ne peuvent pas envoyer d'exercices, cela est dû à un manque d'espace alloué au cours sur le serveur.
J'espère avoir aidé.
atte
Dr Agustín Rico Guzmán
webmaster formation médicale CAR Mexique

**Comment by Moigaston** [comment]
Je vous remercie pour votre aide. J'ai augmenté la taille du cours et il n'y a plus de problème! Merci!
